core/src/main/java/com/linecorp/armeria/common/stream/StreamMessage.java

```diff
@@ -615,6 +615,27 @@ default EventExecutor defaultSubscriberExecutor() {
         return map(function);
     }
 
+    /**
+     * Peeks an error emitted by this {@link StreamMessage} and applies the specified {@link Consumer}.
+     *
+     * <p>For example:<pre>{@code
+     * StreamMessage streamMessage = StreamMessage.aborted(new IllegalStateException("Something went wrong.");
+     * StreamMessage transformed = streamMessage
+     *     .peekError(ex -> {
+     *         assert ex instanceof IllegalStateException;
+     *     })
+     *     .mapError(ex -> new MyDomainException(ex));
```

Comment:
Could you remove this method from the example?

Reply:
Sure!